clients/drcachesim/common/trace_entry.h

```diff
@@ -442,13 +453,38 @@ typedef enum {
      */
     TRACE_MARKER_TYPE_SYSCALL,
 
+    /**
+     * This marker is emitted prior to a system call which is known to block under some
+     * circumstances.  Whether it blocks may depend on the values of parameters or the
```

Comment:
So absence of the marker means it surely doesn't block?

Edit:
From is_maybe_blocking_syscall, it looks like it may still block. Add a one-line caveat here?

Reply:
Added caveats about not being guaranteed to be complete.